server/src/subscriber/subscriber.service.ts

```diff
@@ -183,5 +215,38 @@ export class SubscriberService {
     return validCustomFieldValues.includes(value as CustomFieldValue);
   }
 
+  /**
+   * Convert the uploaded subscriptions object into a format for Handlebars to use in the confirmation email
+   * @param {object} subscriptions - The set of CDs the user is subscribing to
+   * @returns {boolean}
+   */
+  private convertSubscriptionsToHandlebars(subscriptions: object) {
```

Comment:
Try to make a type for this `subscriptions` argument that is stricter than just `object`.